doc/cheatsheet.md

```diff
@@ -320,13 +434,27 @@ I don't find these operators particularly useful in practice (with the exception
   * The reasons for this are very specific to Perl; if you are a true Perl nerd, you can look them up, but you do not need to know them if you just accept that variables need to start with `::` when you apply `use strict`.
   * It is probably a good idea to `use strict` when the variables you define are sufficiently complex; otherwise you're probably okay not using it.
   
-##Plotting with `ni --js`
+##Binary Operations
+In theory, this can save you a lot of space. But I haven't used this in practice.
```

Comment:
The big use case for binary operations is doing stuff like unpacking wav files or other data that would be inconvenient to work with.